lib/compute/zone.js

```diff
@@ -314,11 +314,22 @@ Zone.prototype.createAutoscaler = function(name, config, callback) {
  * @param {object} callback.apiResponse - The full API response.
  *
  * @example
+ * //-
+ * // Create a persistent disk using the latest Ubuntu version as the source image.
+ * //-
  * var config = {
  *   os: 'ubuntu',
  *   sizeGb: 10
  * };
  *
+ * //-
+ * // Create a persistent disk using the latest Ubuntu version from your project as the source image.
```

Comment:
Thank you for adding this example, very nice touch! Can you make sure it fits under 80 chars?